build/run-prow-e2e.sh

```diff
@@ -2,5 +2,66 @@
 # Copyright (c) 2021 Red Hat, Inc.
 # Copyright Contributors to the Open Cluster Management project
 
-# make it always pass to test clusterpool flow
-echo "helloworld"
\ No newline at end of file
+set -e
+
+UI_CURRENT_IMAGE=$1
+
+
+###############################################################################
+# Install oc and Cluster Keeper
+###############################################################################
+echo "Install oc and Cluster Keeper"
+
+echo "Running oc/install..."
+# make oc/install
+echo "Cloning Cluster Keeper"
+git clone https://github.com/open-cluster-management/cluster-keeper.git
+pushd cluster-keeper
+cat > user.env << EOF
+CLUSTERPOOL_CLUSTER=$CLUSTERPOOL_HOST_API
+CLUSTERPOOL_TARGET_NAMESPACE=$CLUSTERPOOL_HOST_NAMESPACE
+CLUSTERCLAIM_GROUP_NAME=$CLUSTERPOOL_GROUP_NAME
+EOF
+export PATH=${PATH}:$(pwd)
+popd
```

Comment:
oc is already available in the Cypress test image, right? And we don't need cluster-keeper.